docs/sources/operator/release-notes.md

```diff
@@ -1,14 +1,14 @@
 ---
 canonical: https://grafana.com/docs/agent/latest/operator/release-notes/
 description: Release notes for Grafana Agent static mode Kubernetes operator
-title: Release notes for Grafana Agent
 menuTitle: Release notes
-aliases:
-- ./upgrade-guide/
```

Comment:
Is the removal of this alias intended?

Reply:
Ooops, no that was a merge conflict resolution thing I missed :-( I'll fix it.